generators/new_collateral/config/parser.go

```diff
@@ -45,23 +44,22 @@ func (parser) ParseCurrentConfig(configFilePath, configFileName string) (types.T
 	}, nil
 }
 
-// ParseExporterMetaData is exported for testing
-func ParseExporterMetaData(tomlConfig types.TransformersConfigForToml) (types.ExporterMetaData, error) {
+func parseExporterMetaData(tomlConfig types.TransformersConfigForToml) (types.ExporterMetaData, error) {
```

Comment:
Updated the tests so that these pieces of functionality are tested through the one exported metohd, `ParseCurrentConfig`. However, this meant that for the error cases, I needed to create new toml files to make sure that the parser behaved properly in those error cases. Totally up for some other ideas on how to handle this!